exclude_gravity: true
decVar_bounds_on: false
udot_bounds_on: true
Nh: 4
h_mpc: 0.015
u_upper_bounds: [87.0, 87.0, 87.0, 87.0, 12.0, 12.0, 12.0]
u_lower_bounds: [-87.0, -87.0, -87.0, -87.0, -12.0, -12.0, -12.0]
x_up_bounds: [2.8973, 1.7628, 2.8973, -0.0698, 2.8973, 3.7525, 2.8973, 2.1750, 2.1750, 2.1750, 2.1750, 2.61, 2.61, 2.61]
x_low_bounds: [-2.8973, -1.7628, -2.8973, -3.0718, -2.8973, -0.0175, -2.8973, -2.1750, -2.1750, -2.1750, -2.1750, -2.61, -2.61, -2.61]
x_entries: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
u_entries: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
base_roll_pitch_yaw: [0.0, 0.0, -90.0]
base_position: [0.0, -0.2, 0.0]
Q_matrix_coeffs: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
R_matrix_coeffs: [200.0, 200.0, 200.0, 200.0, 200.0, 200.0, 200.0]